ios/xcode/FluentUI_common.xcconfig

```diff
@@ -52,3 +52,8 @@ SWIFT_TREAT_WARNINGS_AS_ERRORS = YES
 TARGETED_DEVICE_FAMILY = 1,2
 VERSIONING_SYSTEM = apple-generic
 VERSION_INFO_PREFIX =
+
+ARCHS[sdk=iphonesimulator*] = $(ARCHS_STANDARD) arm64 x86_64
+ARCHS[sdk=iphoneos*] = $(ARCHS_STANDARD) arm64e
+ONLY_ACTIVE_ARCH = NO
```

Comment:
Are people really copying debug bits? This seems like a rather large build time tax for everyone to pay for that?

Reply:
Yep, debug bits are used in several of our consumer's test apps for their own debug builds.